quiz.js

```diff
@@ -8,21 +6,23 @@ function enter(e) {
     growTree()
```

Comment:
You still are not passing the tree object into any function. Maybe instead of calling `growTree()` here you could call `tree()`. Then once you create the object from values in the tree function you can call growTree() and pass the entire object into it.